compio-http/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+[package]
+name = "compio-http"
+version = "0.1.0"
+categories = ["asynchronous", "network-programming"]
+keywords = ["async", "net"]
+edition = { workspace = true }
+authors = { workspace = true }
+readme = { workspace = true }
+license = { workspace = true }
+repository = { workspace = true }
+
+[dependencies]
+compio-buf = { workspace = true, features = ["bytes"] }
+compio-io = { workspace = true }
+compio-runtime = { workspace = true, features = ["time"] }
+compio-net = { workspace = true, features = ["runtime"] }
+compio-tls = { workspace = true }
+
+native-tls = { workspace = true, optional = true }
+rustls = { workspace = true, optional = true }
+rustls-native-certs = { workspace = true, optional = true }
+
+http = "0.2"
+hyper = { version = "0.14", features = ["client", "http1"] }
+send_wrapper = { workspace = true, features = ["futures"] }
+tokio = { workspace = true }
+
+base64 = "0.21"
+cfg-if = { workspace = true }
+encoding_rs = "0.8"
+mime = "0.3"
+serde = "1"
+serde_json = { version = "1", optional = true }
+serde_urlencoded = "0.7"
+thiserror = "1"
+url = "2"
+
+[dev-dependencies]
+compio-macros = { workspace = true }
+hyper = { version = "0.14", features = ["server", "tcp"] }
+
+[features]
```

Comment:
Add a default client feature for possible future server implementation.

Reply:
I doubt if we really want to add a server to this crate...

Reply:
Well you named it http then